api/openapi_server/controllers/auth_controller.py

```diff
@@ -96,15 +98,19 @@ def sign_up(body: dict):
                 raise AuthError({  "message": msg }, 400)
             case 'InvalidPasswordException':
                 msg = "Password did not conform with policy"
+                remove_user(body, True, False)
```

Comment:
since we are using bool args for this method's params, these calls would be more readable as e.g.
```python
remove_user(body, removeDB=True, removeCognito=False)
```

An alternative design could be a integer enum flag, so we could pass one, neither or a binary `OR` as below:
```python
remove_user(body, RemoveUserOptions.RemoveFromDb | RemoveUserOptions.RemoveFromCognito)
```